docs/modules/ROOT/pages/index.adoc

```diff
@@ -18,7 +18,7 @@ ifdef::backend-pdf[]
 Documentation license: <<license, Creative Commons 4.0>>
 endif::[]
 
-> This is the documentation for the Neo4j GraphQL Library version 3.0, authored by the Neo4j GraphQL Team.
+> This is the documentation for the Neo4j GraphQL Library version 3.7, authored by the Neo4j GraphQL Team.
```

Comment:
We clearly missed updating this many times over.. Is it worth stating the version?

Reply:
It would be great to do a find and replace in a workflow build step if the version can be obtained from somewhere else (package.json?)

Reply:
I honestly think we should just get rid of the reference to the version here.

```suggestion
> This is the documentation for the Neo4j GraphQL Library, authored by the Neo4j GraphQL Team.
```